code/script.js

```diff
@@ -1,47 +1,176 @@
 // All the DOM selectors stored as short variables
 const chat = document.getElementById('chat')
+const userInput = document.getElementById('user-input')
+const sendBtn = document.getElementById('send-btn')
+const nameForm = document.getElementById('name-form')
+const startButton = document.getElementById('startButton')
+const startPage = document.getElementById('startPage')
+const inputWrapper = document.getElementById('input-wrapper')
+const messageAudio = new Audio ('./message-pop-alert.mp3');
 
-// Global variables, if you need any, declared here
-
-// Functions declared here
+//FUNCTIONS
 
 // This function will add a chat bubble in the correct place based on who the sender is
 const showMessage = (message, sender) => {
   if (sender === 'user') {
+    console.log("Hello!")
```

Comment:
I think we're suppose to remove all console.logs before handing in? :D I (we) forgets as well heh

Reply:
You're right! There are always bunch of things we forget to do or not to do :)